modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,188 @@
++
+image::model-server-size.png[model server size]
+
+- **Model Route**: There are models that can be consumed only from other containers inside the same OpenShift cluster, here we have the hability to don't make this server available to entities outside our cluster, or to instruct the model server configuration to assign an external route. When we don't expose the model externally through a route, click on the Internal Service link in the Inference endpoint section:
++
+image::figure14_0.png[Inference endpoint]
++
+A popup will display the address for the gRPC and the REST URLs:
++
+image::figure15_0.png[Endpoint URLs] 
+
+- **Token authorization**: In this part of the form we have a helper checkmark to add authorization to a service account that will be created with access to our model server. Only API requests that present a token that has access to the given service account will be able to run the inference service.
+====
+
+. After clicking the **Add** button at the bottom of the form, you will be able to see a new **Model Server** configuration in your project, you can click the **Tokens** column, which will make visible the tokens that you can share with the applications that will consume the inference API.
++
+image::model-server-with-token.png[Model Server with token]
+
+== Deploy The Model
+
+. At the right side of the **Model Server**, we can find the **Deploy Model** button, let's click the **Deploy Model** button, to start filling the **Deploy Model** form:
++
+image::deploy-model-button.png[Deploy Model button]
+
+. Fill the **Deploy Model** from as in the example:
++
+image::deploy-model-form.png[Deploy Model form]
+
+. After clicking the **Add** button at the bottom of the form, you will be able to see a new entry at the **Deployed models** column for your **Model Server**, clicking in the column will eventually show a check mark under the **Status** column:
++
+image::deploy-model-success.png[Deploy model success]
+
+. Observe and monitor the assets created in your OpenShift **iris-project** namespace.
++
+```shell
+oc get routes -n iris-project
+oc get secrets -n iris-project | grep iris-model
+oc get events -n iris-project
+```
++
+image::iris-project-events.png[Iris project events]
++
+[TIP]
+====
+Deploying a **Model Server** triggers a **ReplicaSet** with **ModelMesh**, which attach your model to the inference runtime, and exposes it through a route. Also, notice the creation of a secret with your token.
+====
+
+== Test The Model With CURL
+
+Now that the model is ready to use, we can make an inference using the REST API
+
+. Assign the route to an environment variable in your local machine, so that we can use it in our curl commands
++
+```shell
+export IRIS_ROUTE=https://$(oc get routes -n iris-project | grep iris-model | awk '{print $2}')
+```
+
+. Assign an authentication token to an environment variable in your local machine
```

Comment:
Is it possible to use the token provided in the RHODSAI dashboard as well? You can mention that possibility too in a note or a tip.